decompressed/base/lib/mount_modroot/05_transfer_basefiles

```diff
@@ -4,21 +4,11 @@ upperdir=$1
 overlay_bank=/overlay/bank_2
 
 move_overlay() {
-  #use the www folder as check to know that we should populate the modoverlay (mtd3) and cleanup real overlay (mtd2)
+  #use the config folder as check to know that we should populate the modoverlay (mtd3) and cleanup real overlay (mtd2)
   if [ -d $overlay_bank/etc/config ]; then
 
-    #iterate all bank_2 folders and file to merge content with the one on modoverlay
-    for file in $(find "$overlay_bank/"*/ -xdev | cut -d '/' -f4-); do
-      #create folder if not exist on destination
-      if [[ -d "$overlay_bank/$file" && ! -d "$upperdir/$file" ]]; then
-        mkdir "$upperdir/$file"
-        continue
-      fi
-
-      #if not a dir move the file (and symlink)
-      [ ! -d "$overlay_bank/$file" ] && mv -f "$overlay_bank/$file" "$upperdir/$file"
-
-    done
+    #copy all file from overlay_bank to modoverlay keeping all attributes and links
+    cp -a $overlay_bank/* $upperdir/
```

Comment:
credo che questa finisca in un loop ricorsivo killer che porta al filling di tmd3

Reply:
``bash
find $overlay_bank -mindepth 1 -maxdepth 1 ! -ipath "*/modoverlay" -exec cp -a "{}" "$upperdir/" \;
````